Makefile

```diff
@@ -6,53 +6,46 @@ PORT ?= 3000
 
 .PHONY: dev
 dev:
-	HKNWEB_MODE='dev' pipenv run python ./manage.py runserver $(IP):$(PORT)
+	HKNWEB_MODE='dev' python ./manage.py runserver $(IP):$(PORT)
 
 .PHONY: livereload
 livereload:
-	HKNWEB_MODE='dev' pipenv run python ./manage.py livereload $(IP):$(PORT)
+	HKNWEB_MODE='dev' python ./manage.py livereload $(IP):$(PORT)
 
-setup: pipenv venv migrate
+.PHONY: setup
+setup:
+	python3 -m venv hknweb_dev
```

Comment:
Try printing out instructions?

```make
@echo "When developing, activate the virtualenv with 'source hknweb_dev/bin/activate' so Python can access the installed dependencies."
```

Reply:
Also recommend using a Makefile variable for the actual virtualenv path.

```Makefile
VENV ?= hknweb_dev

...

$(VENV)/bin/activate ...

```

?= allows shell override of variable, but := works equally well.

Reply:
Added Makefile variables for the venv path and the echo instructions